build-patched-kao-kcmo-images.sh

```diff
@@ -56,7 +56,7 @@ fi
 
 function release_image_for_arch() {
      local arch=$1
-     local mirror="https://mirror.openshift.com/pub/openshift-v4/${arch}/clients/ocp"
+     local mirror=$(echo ${MIRROR} | sed "s/$ARCH/$arch/g")
```

Comment:
Maybe rename `ARCH` to `HOST_ARCH` for more clarity? `s/$ARCH/$arch` is not obvious at all :)

I'd include the `/` around `$ARCH` to limit the risk of unexpected matches: `sed "s;/$ARCH/;/$arch/;g"`